.github/workflows/elixir.yml

```diff
@@ -80,11 +105,24 @@ jobs:
           ref: ${{ github.event.inputs.commit_sha }}
       - uses: ./.github/actions/gradle_cache
       - uses: ./.github/actions/elixir_cache
-      - run: cd implementations/elixir && ../../gradlew lint_ockam
+      - run: ../../gradlew build_${{ matrix.mix_project }}
+        working-directory: implementations/elixir
 
-  lint_ockam_kafka:
-    name: Elixir - lint_ockam_kafka
+  test:
+    name: Elixir - test_${{ matrix.mix_project }}
     runs-on: ubuntu-20.04
+    strategy:
```

Comment:
Lets set `fail-fast` to true so that we can all failed test in one workflow run https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#handling-failures

Reply:
Just to clarify, do we want `fail-fast: false` so all sub-jobs will run to completion whether they pass or fail? `true` is the default value.

Reply:
Yes